packages/nuxeo-platform-explorer-package/src/main/resources/install/templates/explorer-deploy/nuxeo.prod

```diff
@@ -0,0 +1,153 @@
+nuxeo.server.hsts.maxage=${env:NUXEO_SERVER_HSTS_MAXAGE:2592000}
+nuxeo.server.hsts.includesubdomains=${env:NUXEO_SERVER_HSTS_INCLUDESUBDOMAINS:false}
+nuxeo.server.hsts.preload=${env:NUXEO_SERVER_HSTS_PRELOAD:false}
+
+# Mail server
+mail.store.protocol=${env:MAIL_STORE_PROTOCOL:pop3}
+mail.store.host=${env:MAIL_STORE_HOST:localhost}
+mail.store.port=${env:MAIL_STORE_PORT:110}
+mail.store.user=${env:MAIL_STORE_USER:anonymous}
+mail.store.password=${env:MAIL_STORE_PASSWORD:secret}
+# Mail transport
+mail.transport.protocol=${env:MAIL_TRANSPORT_PROTOCOL:smtp}
+mail.transport.host=${env:MAIL_TRANSPORT_HOST:localhost}
+mail.transport.port=${env:MAIL_TRANSPORT_PORT:25}
+mail.transport.user=${env:MAIL_TRANSPORT_USER:anonymous}
+mail.transport.password=${env:MAIL_TRANSPORT_PASSWORD:secret}
+mail.transport.usetls=${env:MAIL_TRANSPORT_USETLS:false}
+mail.transport.auth=${env:MAIL_TRANSPORT_AUTH:false}
+# User who will receive mail
+mail.user=${env:MAIL_USER:nobody}
+# Enable debugging output from the javamail classes
+mail.debug=${env:MAIL_DEBUG:false}
+mail.from=${env:MAIL_FROM:noreply@nuxeo.com}
+
+# disable ES usage
+elasticsearch.enabled=false
+
+
+## aws confs mapping
+
+nuxeo.aws.accessKeyId=${env:NUXEO_AWS_ACCESSKEYID:}
+nuxeo.aws.secretKey=${env:NUXEO_AWS_SECRETKEY:}
+nuxeo.aws.region=${env:NUXEO_AWS_REGION:}
+
+
+## s3binaries confs mapping
+
+# mandatory
+nuxeo.s3storage.bucket=${env:NUXEO_S3STORAGE_BUCKET:}
+
+# usually taken from system environement
+nuxeo.s3storage.awsid=${env:NUXEO_S3STORAGE_AWSID:}
+nuxeo.s3storage.awssecret=${env:NUXEO_S3STORAGE_AWSSECRET:}
+nuxeo.s3storage.awstoken=${env:NUXEO_S3STORAGE_AWSTOKEN:}
+nuxeo.s3storage.region=${env:NUXEO_S3STORAGE_REGION:}
+
+# optional
+nuxeo.s3storage.cacheminage=${env:NUXEO_S3STORAGE_CACHEMINAGE:60}
+nuxeo.s3storage.cachesize=${env:NUXEO_S3STORAGE_CACHESIZE:100MB}
+nuxeo.s3storage.cachecount=${env:NUXEO_S3STORAGE_CACHECOUNT:10000}
+nuxeo.s3storage.connection.max=${env:NUXEO_S3STORAGE_CONNECTION_MAX:50}
+nuxeo.s3storage.connection.retry=${env:NUXEO_S3STORAGE_CONNECTION_RETRY:3}
+nuxeo.s3storage.connection.timeout=${env:NUXEO_S3STORAGE_CONNECTION_TIMEOUT:50000}
+nuxeo.s3storage.socket.timeout=${env:NUXEO_S3STORAGE_SOCKET_TIMEOUT:50000}
+nuxeo.s3storage.endpoint=${env:NUXEO_S3STORAGE_ENDPOINT:}
+nuxeo.s3storage.pathstyleaccess=${env:NUXEO_S3STORAGE_PATHSTYLEACCESS:false}
+nuxeo.s3storage.accelerateMode=${env:NUXEO_S3STORAGE_ACCELERATEMODE:false}
+nuxeo.s3storage.bucket_prefix=${env:NUXEO_S3STORAGE_BUCKET_PREFIX:}
+nuxeo.s3storage.crypt.keystore.file=${env:NUXEO_S3STORAGE_CRYPT_KEYSTORE_FILE:}
+nuxeo.s3storage.crypt.keystore.password=${env:NUXEO_S3STORAGE_CRYPT_KEYSTORE_PASSWORD:}
+nuxeo.s3storage.crypt.key.alias=${env:NUXEO_S3STORAGE_CRYPT_KEY_ALIAS:}
+nuxeo.s3storage.crypt.key.password=${env:NUXEO_S3STORAGE_CRYPT_KEY_PASSWORD:}
+nuxeo.s3storage.crypt.serverside=${env:NUXEO_S3STORAGE_CRYPT_SERVERSIDE:false}
+
+nuxeo.s3storage.useDirectUpload=${env:NUXEO_S3STORAGE_USEDIRECTUPLOAD:false}
```

Comment:
let's not use direct upload or direct download - this is supported in NCO/NCP, but not sure that we have what is needed on the IT Hosting side yet and anyway for the Explorer we probably do not care.

Reply:
removed